docs/PUBLISHING.md

```diff
@@ -12,11 +12,32 @@ Look to [this helpful document](http://designsystem.morningstar.com/about/versio
 
 ### Releasing a new version of EDS
 
-Note: the following steps are not fully baked yet and there's quite a few things to iron out. This is the workflow as it currently stands but is likely going to change.
-
-1. When a new release is coming up, a `release` branch is created from `develop` (i.e. `release-1.1`). In this branch, only bugfixes and metadata commits are permitted in order to prepare for releasing a new version of the design system. Update the changelog to represent changes in latest version.
-2. Merge release branch into `main` when the release is ready
-3. Run `npm version [major|minor|patch] -m "tag message goes here"` to tag the release. [npm version](https://docs.npmjs.com/cli/version.html) increments the version number in `package.json` in addition to creating a git tag for the new release.
-4. Run `git tag -a v[version] -m "tag message goes here"` to tag the design system assets for the new release.
-5. Push the changes to the `main` branch.
-6. Communicate the changes via all appropriate channels. TODO determine communication strategy for new release
+1. Confirm that all checks are green on CI.
+2. Run `git checkout main`.
+3. Run
+
+```
+yarn release
+
+# or, if there are breaking changes
+yarn release:breaking
+```
+
+We use [standard-version](https://github.com/conventional-changelog/standard-version) to increment the version number in `package.json`, create a git tag for the new release, and update `CHANGELOG.md` based on the commit log. The package is not published, yet.
+
+4. Run the last command output by `standard-version`. It will look something like: `git push --follow-tags origin <branch> && npm publish`.
+5. Communicate the changes via all appropriate channels. <TODO: determine communication strategy for new release>
+
+**Before the first time you publish**, make sure to:
+
+- set up Two Factor Authentication for your npm account
+- run `npm login` in your terminal to generate an access token for publishing
+
```

Comment:
Could we move this above the main instructions? I actually ran into this at the bee workshop where they had the hot tips after the main instructions, so I didn't read them until I was done with that section and had found out the hard way.